src/Features/Core/Portable/Completion/CompletionItem.cs

```diff
@@ -51,7 +52,9 @@ public sealed class CompletionItem : IComparable<CompletionItem>
         /// The text used to determine the order that the item appears in the list.
         /// This is often the same as the <see cref="DisplayText"/> but may be different in certain circumstances.
         /// </summary>
-        public string SortText { get; }
+        public string SortText => _sortText ?? DisplayText;
+
+        internal bool HasDifferentSortText => _sortText != null;
```

Comment:
why do we need this?  Why can't LSP make this determination when examining the completion items?

Reply:
also, i can't see any use of this in our code.

Reply:
It duplicates the filter implementation of the same, but it is no longer used in the handler so I will remove it.